react/.env

```diff
@@ -1,2 +1,3 @@
 IMAGE_INLINE_SIZE_LIMIT=1000000
-REACT_APP_API_URL=https://api.paybutton.org
\ No newline at end of file
+REACT_APP_API_URL=https://api.paybutton.org
+REACT_APP_API_PRICING_URL=http://localhost:3000
```

Comment:
Why is this set to localhost by default and the main URL using api.paybutton.org by default?

Reply:
Let's set this to be the same as the main API host for now. Until we run into scalability issues, it should be fine.